### DISCUSSION TITLE
Update to 20220904 from latest image (202207101) is not possible

### DISCUSSION THREAD

**Discussion by sp300b** [rern/rAudio#1008] [Fixed Issues] [answered]
Hi, 
Please check- there is no option to update from image file 
![20220909-no](https://user-images.githubusercontent.com/94806458/188327328-78eaa07b-ab1c-4eae-9ed6-131031bc43d9.PNG)
![20220909-system](https://user-images.githubusercontent.com/94806458/188327417-5355f535-6873-4412-8d88-746f8aa51452.PNG)

**Comment by cmh714** [comment, marked as answer]
Set the date to earlier than today....in this file with theis format:  /srv/http/data/addons/r1
20220904

**Reply by mvught** [reply]
if i adjust the file then the button is grayed out, refresh or reboot but now result
<img width="703" alt="Screenshot 2022-09-05 at 09 43 01" src="https://user-images.githubusercontent.com/14079626/188395504-849a4cc1-6b84-4d20-b691-44724400b571.png">

**Reply by mvught** [reply]
Terminal Result is ok
<img width="459" alt="Screenshot 2022-09-05 at 09 47 39" src="https://user-images.githubusercontent.com/14079626/188396479-d082a790-6569-495d-9058-19798f4b9960.png">

**Reply by mvught** [reply]
fixed sorry date had to be in the past! 
20220906 ;)

**Reply by sp300b** [reply]
Fixed this way, Thanks for the tip!
